### ISSUE TITLE
How to get the path of the original svg file for templating?

### ISSUE THREAD

**Issue by Voo2** [jkphl/svg-sprite#202]
Is it possible to get the relative path + filename of the original svg file for using it templates?

**Comment by jkphl** [comment]
@Voo2 Please describe your exact situation and why you would need the original SVG path in your template (and maybe see #196 as well). Thanks!

**Comment by Voo2** [comment]
I have very special requirements in my current project. I have to create a svg sprite, a css file with selectors targeting the svg sprite and an addition css file which contains the path to the stand alone svg files instead to the svg sprite. The 2nd css file should be a kind of fallback. To create the svg sprite and the 1st css file is not a problem. But I don't know how to insert the path to the original svg file into the 2nd css file. 

If necessary to understand my problem I could attach my gulpfile.js and/or my template files. But they are very special, which is why I believe it will only be unnecessarily confusing.

**Comment by Voo2** [comment]
Is there really no way?

**Comment by jkphl** [comment]
@Voo2 No, *svg-sprite* doesn't keep any references to the original source SVG files for templating purposes. Please keep in mind that it's *svg-sprite*'s first and foremost task to create a sprite and not render stuff for the original SVGs.

Another option you might have though is take a look at [node-iconizr](https://github.com/jkphl/node-iconizr/tree/dev) (dev branch!) respectively [gulp-iconizr](https://github.com/jkphl/gulp-iconizr). It's based on SVG based on *svg-sprite* but always renders 4 different stylesheets:

* SVG sprite
* SVG single files (optimized)
* PNG sprite
* PNG single files (optimized)

I'm sorry it's fairly undocumented, but I just wrote the Gulp plugin this weekend (after a 2 years break) and it works! Maybe you can use that output?
